data_steward/utils/participant_summary_requests.py

```diff
@@ -40,6 +49,18 @@
 to the Curation naming convention in the `get_site_participant_information` function
 """
 
+FIELDS_OF_INTEREST_FOR_DIGITAL_HEALTH = [
+    'participantId', 'digitalHealthSharingStatus'
+]
+
+BASE_URL = 'https://{api_project_id}.appspot.com/rdr/v1/ParticipantSummary'
+
+# Retry parameters
+STATUS_FORCELIST = [500, 502, 503, 504]
+MAX_RETRIES = 5
+BACKOFF_FACTOR = 0.5
+MAX_TIMEOUT = 60
```

Comment:
From the end of the second paragraph [here](https://docs.python-requests.org/en/master/user/advanced/#timeouts), I would suggest setting the MAX_TIMEOUT to 62 instead of 60.